tests/prod/prod_sanity_suite_test.go

```diff
@@ -242,4 +303,15 @@ func TestVirtualDevice2PortCreateConnection_DIGP(t *testing.T) {
 	terraform.InitAndApply(t, terraformOptions)
 	output := terraform.Output(t, terraformOptions, "port_connection_id")
 	assert.NotNil(t, output)
+
+	terraform.Apply(t, terraformOptions)
```

Comment:
This shouldn't need to be here because it is already being done earlier in method with `InitAndApply`. Why is the redundant `Apply` in place?